### PR TITLE
Add character n to banner

### LINKED ISSUE #497: Add character: <n> to font: <banner>
*Character* : <n>
*Font* : <banner>

### PR DESCRIPTION
**Pull request by ben-dh3**
Added character 'n' to font 'banner'

### Referenced Issues

closes #497

### CODE REVIEW

**Comment by AnshumanMahato** [comment]
@ben-dh3 please request for assignment before opening up a PR. Otherwise, it creates a lot of confusion for use. I hope you understand. So please ensure this for further contributions. 😊

File: Fonts/banner/banner.h [hunk 1]
```diff
@@ -427,17 +427,17 @@ class Banner : public Fonts
...
     */
 
-    /*
-        char **n()
-        {
-            char **character = getCharGrid(ROWS,COLS);
+    
+    char **n()
+    {
+        char **character = getCharGrid();
```
Comment (AnshumanMahato) [inline Fonts/banner/banner.h:433]: pass the no. of rows and columns that you used to make the character

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]
Adjust your grid in a way so that it aligns with the baseline.
Currently, this is happening...
image

File: Fonts/banner/banner.h (hunk 1 above)
Comment (ben-dh3) [reply Fonts/banner/banner.h]: I have added the rows and columns used. I hope that works! Thanks

Comment (AnshumanMahato) [reply Fonts/banner/banner.h]: @ben-dh3, well, the spacing to the right is resolved. But it is still not aligned with the bottom.
image

From what I can see... taking two extra rows and leaving the top 2 rows empty should do the trick.

Comment (ben-dh3) [reply Fonts/banner/banner.h]: That should work now!

**Review by AnshumanMahato — APPROVED** [review]

**Pull request merged**